ccmlib/scylla_cluster.py

```diff
@@ -141,7 +144,12 @@ def start_nodes(self, nodes=None, no_wait=False, verbose=False, wait_for_binary_
                 for node, _, _ in started:
                     if old_node is not node:
                         old_node.watch_log_for_alive(node, from_mark=mark)
-
+                        def check_gossip():
```

Comment:
It will better to create a separate method in Node class:
```
    def wait_for_gossip(self, timeout=30):
        if not wait_for(lambda: self.address() in self.nodetool('gossipinfo')[0], step=0.001, timeout=timeout,
                        text=f"wait to gossip for {self.address()}"):
            raise RuntimeError(f"{self.name} wasn't found in {self.name} gossip")
```

Reply:
BTW this wrong logic, we need to verify against other node, not the node vs. itself.

a node gossip would always show itself...